client/my-sites/email/email-providers-comparison/in-depth/comparison-list.tsx

```diff
@@ -51,6 +51,12 @@ const ComparisonList = ( {
 							<LearnMoreLink url={ emailProviderFeatures.supportUrl } />
 						</div>
 
+						{ emailProviderFeatures.badge && (
+							<div className="email-providers-in-depth-comparison-list__badge">
+								{ emailProviderFeatures.badge }
+							</div>
+						) }
+
 						<Button
 							className="email-providers-in-depth-comparison-list__button"
```

Comment:
Why aren't we disabling this Select button the same way as we are in the comparison table? The button is not going to work for Google Workspace monthly.

Reply:
That is a mistake, I'll fix that.